src/routers/appRouters.js

```diff
@@ -16,7 +16,12 @@ router.get('/', async (req, res) => {
 	try {
 		const query = 'name' in req.query ? req.query.name : ''
 		const apps = await db.loadApps(query).catch(e => { throw e })
-		const ipsTotal = apps.map(app => app.getIps().length).reduce((a, b) => a + b, 0)
+		let ipsTotal = 0
+		if (Array.isArray(apps)) {
```

Comment:
@gutorc92, why would `apps` be an array? is it because of older version compatibility issues?

Reply:
App resource can return a object or an array:

``` /app?name=teste``` return a single app as a object

``` /app ``` return all apps as an array